Tchap/Modules/Room/Members/RoomParticipantsViewController.m

```diff
@@ -556,10 +561,12 @@ - (void)setKeyboardHeight:(CGFloat)keyboardHeight
     super.keyboardHeight = keyboardHeight;
     
     // Update addParticipants button position with animation
+    MXWeakify(self);
```

Comment:
weak self strong self dance is not necessary here. Self is retained during the animation duration only.